Source/SwiftLintFramework/Extensions/SyntaxMap+SwiftLint.swift

```diff
@@ -13,11 +13,11 @@ extension SyntaxMap {
     /// Returns array of SyntaxTokens intersecting with byte range
     ///
     /// - Parameter byteRange: byte based NSRange
-    internal func tokensIn(_ byteRange: NSRange) -> [SyntaxToken] {
+    internal func tokens(inByteRange byteRange: NSRange) -> [SyntaxToken] {
```

Comment:
I like how this now express that it's a byte range - I've always had to lookup in the header